contracts/AssetGovernance.sol

```diff
@@ -53,34 +44,19 @@ contract AssetGovernance is ReentrancyGuard {
     /// @notice Address that collects listing payments
     address public treasury;
 
-    /// @notice AccountIndex that collects listing payments
-    uint32 public treasuryAccountIndex;
```

Comment:
why `treasuryAccountIndex` is deleted? according to the comments, it is not related with AMM swap.